### PR TITLE
List item styling

### LINKED ISSUE #25: Lookbook/basket page: Shows (unstyled) list of "liked" looks
Uses JSON mapping to parse a list of looks to show on "lookbook page"

### LINKED ISSUE #26: Lookbook/basket page: Bottom of page has a floating buy all button
Functionality optional for MVP

### LINKED ISSUE #27: Styling of Lookbook/Basket page: Cards for items

### LINKED ISSUE #28: Styling of Lookbook/Basket page: Card Elements
Each card has:

Picture
Name
Shortened description
LInks to product info page
Price

### LINKED ISSUE #30: Lookbook/Basket page has link to main swiping page

### PR DESCRIPTION
**Pull request by kaihoffman**
Re-styled the buy button on the basket/lookbook page. 

Closes #30 
Closes #26 
Closes #27 
Closes #25
Closes #28

### CODE REVIEW

**Review by k-ankobia — APPROVED** [review]
Looks Good

**Review by nkhil — APPROVED** [review]

File: cypress/integration/visit_lookbook_page_test.spec.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    cy.url().should('eq', 'http://localhost:1234/lookbook')
+  })
+
+  it('can direct the user from look page to basket', () => {
+    cy.visit('http://localhost:1234/')
+    cy.get('#lookbook').click()
+    cy.get('#lookpage').click()
+    cy.url().should('eq', 'http://localhost:1234/')
+  })
+
```
Comment (ibrahimbutt) [inline cypress/integration/visit_lookbook_page_test.spec.js:14]: You should be using Prettier to format your files.

File: src/Basket/index.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+`
+const FooterStyling = styled.div`
+  display: grid;
+  grid-template-columns: repeat(11, 2fr);
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 70px;
+  // background-color: #000000;
```
Comment (ibrahimbutt) [inline src/Basket/index.js]: Don't push commented code.

File: src/Basket/index.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+  margin: 0 auto;
+`
+
+
+
+const ArrowMargin = {
+  marginLeft: '25px',
+}
+
+const StyledLookPageLink = styled(LookPageLink)`
```
Comment (ibrahimbutt) [inline src/Basket/index.js]: What is the purpose of this styled component? It has no styling.

**Review by ibrahimbutt — CHANGES_REQUESTED** [review]
Have a look at the comments I left. Let me know when you've made the changes, and I'll review them.

Don't resolve the comments.

**Comment by kaihoffman** [comment]
Should be fine now?

**Review by ibrahimbutt — APPROVED** [review]

**Pull request merged**